Rakefile

```diff
@@ -2,5 +2,11 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'rubocop/rake_task'
 
 Rails.application.load_tasks
+
+  desc 'Runs rubocop rake task'
+  task :static_analysis do
+    RuboCop::RakeTask.new
```

Comment:
Can we run brakeman here as well? http://brakemanscanner.org/docs/rake/  We should update the `desc` as well if we do this.